app/src/main/java/com/team41/boromi/book/BorrowedFragment.java

```diff
@@ -0,0 +1,65 @@
+package com.team41.boromi.book;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.team41.boromi.R;
+
+/**
+ * A simple {@link Fragment} subclass. Use the {@link BorrowedFragment#newInstance} factory method
+ * to create an instance of this fragment.
+ */
+public class BorrowedFragment extends Fragment {
```

Comment:
The borrowed class should have 3 subtabs, similar to the OwnedFragment, but this can probable be moved off to #23

Reply:
I am moving the rest of the tabs into a new issue, the implementation with fragments for buttons and tabs is different so I didn't see any value in creating buttons in all the pages just to be replaced later. However, if you are working on another page and would like the buttons there it is really easy to just add a button to the XML and add the onclick listener that initializes the general fragment.